src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -271,8 +278,24 @@ endif ()
 if(ENABLE_SHL_FOR_CPU)
     target_link_libraries(${TARGET_NAME} PRIVATE shl)
 endif()
+set(KLEIDIAI_INCLUDE_DIRS
+        thirdparty/kleidiai/
+        thirdparty/kleidiai/kai/
+        thirdparty/kleidiai/kai/ukernels/
+        thirdparty/kleidiai/kai/ukernels/matmul/
+        thirdparty/kleidiai/kai/ukernels/matmul/matmul_clamp_f32_qai8dxp_qsi4cxp/
+        thirdparty/kleidiai/kai/ukernels/matmul/matmul_clamp_f32_qsi8d32p_qsi4c32p/
+        thirdparty/kleidiai/kai/ukernels/matmul/matmul_clamp_f32_qai8dxp_qsi4c32p/
+        thirdparty/kleidiai/kai/ukernels/matmul/pack/
+)
+include_directories(SYSTEM INTERFACE ${KLEIDIAI_INCLUDE_DIRS})
+target_include_directories(${TARGET_NAME} SYSTEM PRIVATE $<TARGET_PROPERTY:kleidiai,KLEIDIAI_INCLUDE_DIRS>)
```

Comment:
there is no such property `KLEIDIAI_INCLUDE_DIRS` in cmake.

you need to fix `kleidiai`'s library cmake files, so target will have public include directories as configured (via `target_include_directories`) and you don't have to list all those dirs in `KLEIDIAI_INCLUDE_DIRS`